shaders/src/ambient_occlusion.fs

```diff
@@ -141,12 +140,13 @@ float computeSpecularAO(const highp vec2 uv, float NoV, float visibility, float
 #if defined(BLEND_MODE_OPAQUE) || defined(BLEND_MODE_MASKED)
 
 #if SPECULAR_AMBIENT_OCCLUSION == SPECULAR_AO_SIMPLE
+    // TODO: Should we even bother computing this when screen space bent normals are enabled?
```

Comment:
I was also wondering about that. But since it is a material parameter, this is the responsibility of the user, and since it doesn't do anything "wrong", I feel we should leave it.